.github/workflows/lint.yml

```
@@ -26,3 +26,1591 @@ jobs:
         with:
           args: --timeout=5m
           version: v2.0
+repos:
+
+-   repo: https://github.com/pre-commit/pre-commit-hooks
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)